spring-cloud-gcp-autoconfigure/src/main/java/org/springframework/cloud/gcp/autoconfigure/storage/GcpStorageAutoConfiguration.java

```diff
@@ -53,20 +54,22 @@
 @ConditionalOnProperty(value = "spring.cloud.gcp.storage.enabled", matchIfMissing = true)
 @EnableConfigurationProperties({GcpProperties.class, GcpStorageProperties.class})
 @Import(GoogleStorageProtocolResolver.class)
-public abstract class GcpStorageAutoConfiguration { //NOSONAR squid:S1610 must be a class for Spring
+public abstract class GcpStorageAutoConfiguration {
+	private GcpProjectIdProvider gcpProjectIdProvider; //NOSONAR squid:S1610 must be a class for Spring
 
 	@Bean
 	@ConditionalOnMissingBean
-	public static Storage storage(CredentialsProvider credentialsProvider,
-			GcpStorageProperties gcpStorageProperties,
-			GcpProjectIdProvider projectIdProvider) throws IOException {
```

Comment:
Why are you removing the `projectIdProvider` parameter?

Reply:
Forgot to replace it, but I can see @dzou has setup a replacement already.